File: Programming Basics with JS/T03 - Conditional Statements/lecture/07. Area of Figures.js
function areaOfFigures(input) {
    let figure = (input[0]);

    if (figure === "square") {
        let side = Number(input[1]);
        console.log((side * side).toFixed(3));

    } else if (figure === "rectangle") {
        let sideA = Number(input[1]);
        let sideB = Number(input[2]);
        console.log((sideA * sideB).toFixed(3));

    } else if (figure === "circle") {
        let radius = Number(input[1]);
        console.log((Math.PI * Math.pow(radius, 2)).toFixed(3));

    } else if (figure === "triangle") {
        let side = Number(input[1]);
        let heigh = Number(input[2]);
        console.log(((side * heigh) / 2).toFixed(3));
    }
}
areaOfFigures(["square", "5"]);

File: Programming Basics with Go/T03 - Conditional Statements/lecture/07. Area of Figures.go
package main

import (
	"fmt"
	"math"
)

func main (){
	var figureType string
	fmt.Scanln(&figureType)

	var firstParameter float32
	fmt.Scanln(&firstParameter)

	var area float32
	if figureType == "square" {
		area = float32(math.Pow(float64(firstParameter), 2))
	} else if figureType == "rectangle" {
		var secondParameter float32
		fmt.Scanln(&secondParameter)
		area = firstParameter * secondParameter
	} else if figureType == "circle" {
		area = float32(math.Pi * math.Pow(float64(firstParameter), 2))
	}else if figureType == "triangle" {
		var secondParameter float32
		fmt.Scanln(&secondParameter)
		area = firstParameter * secondParameter / 2
	}
	fmt.Printf("%.3f", area)
}